tests/README.md

```diff
@@ -8,3 +8,23 @@ You can override the location of the binary using `DRAGONFLY_HOME` environment v
 
 to run pytest, run:
 `pytest -xv pytest`
+
+
+# Integration tests
+To simplify running integration test each package should have its own Dockerfile. The Dockerfile should contain everything needed in order to test the package against Drafongly. Docker can assume Dragonfly is running on localhost:6379.
```

Comment:
comment really: some tests will require restarting the process. then your assumption won't hold.

Reply:
Those tests will need to start and stop Dragonfly on their own. Which is something I was trying to avoid for simplicity.